pulser/sequence.py

```diff
@@ -218,6 +221,22 @@ def available_channels(self) -> dict[str, Channel]:
                     and (ch.basis == 'XY' if self._in_xy else ch.basis != 'XY')
                     }
 
+    @property
+    def magnetic_field(self) -> np.ndarray:
+        """The magnetic field acting on the array of atoms.
+
+        The magnetic field vector is defined on the reference frame of the
+        atoms in the Register (with the z-axis coming outside of the plane).
+
+
+        Note:
+            Only defined in "XY Mode", the default value being (0, 0, 1) G.
```

Comment:
```suggestion
            Only defined in "XY Mode", the default value being (0, 0, 30) G.
```

Reply:
Thanks for spotting it, I knew writing it in the docstring would backfire, I just didn't expect it to be this early haha